lib/types/src/index.ts

```diff
@@ -9,7 +9,7 @@ const packageJsonPath = path.resolve(__dirname, '../package.json')
 const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf8'))
 const version: string = packageJson.version
 
-// uses Symbol.for, not Symbol, so they're compatible across different
+// uses Symbol.for, not Symbol, so that they're compatible across different
```

Comment:
question: with the additional commits you have added into the commit message footer - 
ie. 
```
feat: add validation errors to compatibility test in types package
refactor: better name than "something"
feat: loosen task and hook instance checks
```
will this mean release please will generate changelogs for these multiple features/refactor, from what I understand that are just a reference to the old commits that we've made in `main`? unfortunately the [doc](https://github.com/googleapis/release-please#what-if-my-pr-contains-multiple-fixes-or-features) doesn't make it very clear, nonetheless I don't think we want release-please to add these three to the changelog

Reply:
like those docs say, release-please [parses](https://github.com/googleapis/release-please/blob/f75e49f6b751347456746d9c4068191d340c8e1e/src/commit.ts#L223-L288) all the text in the git footer that look like commit messages as if they are additional conventional commits. these three commits are old commits that have already been committed to main but were missed by release-please because of https://github.com/googleapis/release-please/issues/1533. these changes are all from #245 though so why wouldn't we want to include them in the changelog?

Reply:
ah right, they're the commits that are missed, that makes sense - thanks for explaining!